### PR TITLE
PROC-1294 Add payments embed to the order resource

### PR DESCRIPTION
**Pull request by Smitsel**

### CODE REVIEW

File: source/reference/v2/orders-api/get-order.rst [hunk 1]
```diff
@@ -36,6 +36,13 @@ available.
 
      - Set this to ``true`` to retrieve a test mode order.
 
+Embedding of related resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This endpoint also allows for embedding additional information by appending the following values via the ``embed``
+query string parameter.
+
+* ``payments`` Include any :doc:`payments </reference/v2/payments-api/get-payment>` created for the order.
```
Comment (mvdpanne) [inline source/reference/v2/orders-api/get-order.rst]: I'd prefer `all` above `any`.

File: source/reference/v2/orders-api/get-order.rst [hunk 2]
```diff
@@ -645,6 +652,44 @@ Response
...
+                {
+                    "resource": "payment",
+                    "id": "tr_ncaPcAhuUV",
+                    "mode": "live",
+                    "createdAt": "2018-09-07T12:00:05+00:00",
+                    "amount": {
+                        "value": "1027.99",
+                        "currency": "EUR"
+                    },
+                    "description": "Order Order #1337 (Lego cars)",
```
Comment (ndijkstra) [inline source/reference/v2/orders-api/get-order.rst]: Double "Order"

File: source/reference/v2/orders-api/get-order.rst (hunk 1 above)
Comment (Smitsel) [reply source/reference/v2/orders-api/get-order.rst]: √

File: source/reference/v2/orders-api/get-order.rst (hunk 2 above)
Comment (Smitsel) [reply source/reference/v2/orders-api/get-order.rst]: √

**Pull request merged**